AutomatedTesting/Levels/Base/Base.prefab

```diff
@@ -17,7 +17,16 @@
             },
             "Component_[14126657869720434043]": {
                 "$type": "EditorEntitySortComponent",
-                "Id": 14126657869720434043
```

Comment:
Is it intended to check this change in? Can it affect any of the other levels which load the base prefab?

Reply:
There wasn't a Base.prefab, I was told to create one.